packages/frontend-api/src/Resources/config/graphql-types/SettingsDecorator.types.yaml

```diff
@@ -32,3 +32,15 @@ SettingsDecorator:
                 type: Boolean!
                 resolve: "@=query('heurekaEnabledQuery')"
                 description: "Returns true if Heureka is available for the current domain"
+            termsAndConditionsArticleUrl:
+                type: String
+                resolve: "@=query('termsAndConditionsArticleUrlQuery')"
+                description: ""
```

Comment:
Can you please fill in the descriptions for special articles?

Reply:
Yes. Thanks for pointing it out.